charts/argo-workflows/templates/_helpers.tpl

```diff
@@ -97,6 +97,13 @@ app.kubernetes.io/component: {{ .component }}
 {{- end }}
 {{- end }}
 
+{{/*
+Create the name of the controller configMap
+*/}}
+{{- define "argo-workflows.controller.config-map.name" -}}
+{{- .Values.controller.configMap.name | default (printf "%s-%s" (include "argo-workflows.controller.fullname" .) "configmap") | trunc 63 | trimSuffix "-" -}}
```

Comment:
I wonder how Argo Workflows controller refers the ConfigMap if `.Values.controller.configMap.name` is configured. 🤔

Reply:
Ah this define is included in `charts/argo-workflows/templates/controller/workflow-controller-deployment.yaml` as well.